### ISSUE TITLE
ome data format import

### ISSUE THREAD

**Issue by manerotoni** [tischi/fiji-plugin-bigDataProcessor2#60]
Dear Tischi, 
thanks for the nice tool. I was wondering is you can create an import an ome.xml file created from a  Viventis microscope. 

I attach the ome file here. The tiff files are multi-page -Z stacks one file per time point and channel. 

[ome-tiff.companion.zip](https://github.com/tischi/fiji-plugin-bigDataProcessor2/files/3676055/ome-tiff.companion.zip)

Thanks

Antonio

**Comment by tischi** [comment]
1. What happens if you open this in ImageJ?
2. Could I also get the images, such that I can test?

**Comment by manerotoni** [comment]
Yes I will make you owncloud folder for it. Bioformat opens it as imagej virtual stack. In fact it could also be useful to have a function like 'open current image in BigDataTools'. With this you avoid creating a lot of porting between formats and still use the virtual stacks.

**Comment by tischi** [comment]
>  like 'open current image in BigDataTools'. With this you avoid creating a lot of porting between formats and still use the virtual stacks.

Yes, you are right. In fact, we started working on this. I need to try to make this work.

**Comment by tischi** [comment]
Would you have some time to actually contribute to this in terms of Java code? I could point you to the pieces of code.

**Comment by manerotoni** [comment]
Yes I could work on it. It is also a good occasion to learn some new classes :-)

**Comment by tischi** [comment]
To get into the topic I recommend reading this thread:
- https://forum.image.sc/t/wrap-imageplus-virtualstack-into-imglib2/4154/31

This code is interesting:
- https://github.com/imglib/imglib2-ij/blob/master/src/main/java/net/imglib2/img/VirtualStackAdapter.java

In my code (BigDataProcessor2) we need to implement a version of below class for an ImagePlus:
- https://github.com/tischi/fiji-plugin-bigDataProcessor2/blob/master/src/main/java/de/embl/cba/bdp2/loading/ImageLoader.java#L30

In fact, I think it could be relatively straight-forward, as `VirtualStackAdapter.java` is already containing all the relevant code.
